README.md

```diff
@@ -1,15 +1,9 @@
-# Redis Connection
+# Redis Connection Manager
 
-A ***Global Redis Connection*** that can be used anywhere in your app
+A ***Global Redis Connection Manager*** that can be used anywhere in your app
 and closed ***once*** at the end of tests.
 
-[![Build Status](https://travis-ci.org/dwyl/redis-connection.svg)](https://travis-ci.org/dwyl/redis-connection)
-[![Code Climate](https://codeclimate.com/github/dwyl/redis-connection/badges/gpa.svg)](https://codeclimate.com/github/dwyl/redis-connection)
-[![codecov.io](http://codecov.io/github/dwyl/redis-connection/coverage.svg?branch=master)](http://codecov.io/github/dwyl/redis-connection?branch=master)
-[![Dependency Status](https://david-dm.org/dwyl/redis-connection.svg)](https://david-dm.org/dwyl/redis-connection)
-[![devDependency Status](https://david-dm.org/dwyl/redis-connection/dev-status.svg)](https://david-dm.org/dwyl/redis-connection#info=devDependencies)
-<!-- [![HitCount](https://hitt.herokuapp.com/nelsonic/redis-connection.svg)](https://github.com/dwyl/redis-connection) -->
-[![npm](https://img.shields.io/npm/v/redis-connection.svg)](https://www.npmjs.com/package/redis-connection)
+[![npm](https://img.shields.io/npm/v/redis-connection-manager.svg)](https://www.npmjs.com/package/redis-connection-manager)
```

Comment:
@codeslayer1 you've pushed changes on this PR which pertain to your forked package. Please revert these before fixing failing tests. 👍